datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/resolvers/search/SearchUtils.java

```diff
@@ -9,9 +9,41 @@
   private SearchUtils() {
   }
 
+  /**
+   * Entities that are searched by default in Search Across Entities
+   */
   public static final List<EntityType> SEARCHABLE_ENTITY_TYPES =
-      ImmutableList.of(EntityType.DATASET, EntityType.DASHBOARD, EntityType.CHART, EntityType.MLMODEL,
-          EntityType.MLMODEL_GROUP, EntityType.MLFEATURE_TABLE, EntityType.DATA_FLOW, EntityType.DATA_JOB,
-          EntityType.GLOSSARY_TERM, EntityType.TAG, EntityType.CORP_USER, EntityType.CORP_GROUP, EntityType.CONTAINER,
+      ImmutableList.of(
+          EntityType.DATASET,
+          EntityType.DASHBOARD,
+          EntityType.CHART,
+          EntityType.MLMODEL,
+          EntityType.MLMODEL_GROUP,
+          EntityType.MLFEATURE_TABLE,
+          EntityType.DATA_FLOW,
+          EntityType.DATA_JOB,
+          EntityType.GLOSSARY_TERM,
+          EntityType.TAG,
+          EntityType.CORP_USER,
+          EntityType.CORP_GROUP,
+          EntityType.CONTAINER,
           EntityType.DOMAIN);
+
+  /**
+   * Entities that are part of autocomplete by default in Auto Complete Across Entities
+   */
+  public static final List<EntityType> AUTO_COMPLETE_ENTITY_TYPES =
```

Comment:
does container and domain not support autocomplete?

Reply:
Currently - no